ui/public/styles/main.less

```diff
@@ -67,6 +67,44 @@
   line-height: 0;
 }
 
+@import "mp-tabsets.less";
```

Comment:
@david-martin  I intend to pull any styling into logical separate files, components, views, mixins etc.
I am gonna go with the `mp` prefix for our styling, components unless you have a better suggestion?

Reply:
@sedroche would `mcp` as a prefix make more sense?

Reply:
From what I see most go for 2 character prefixes mimicking angulars `ng-*`.
patternfly use `pf-*`.
`mp` as "mobile panel" seems to fit that convention.

Reply:
ok